apiserver/examples/upload-download-file/main.go

```diff
@@ -228,11 +231,11 @@ func genURL(
 		klog.Errorf("[Error] send genMultipartURL request error %s", err)
 		return GenChunkURLResult{}, err
 	}
+	defer resp.Body.Close()
```

Comment:
@dayuy @0xff-dev So this is only a client side example to upload folders ,right?

Reply:
Yes. Our API can only do one file upload, to upload a directory you need the client to upload each file according to the directory structure.